cognee/tests/integration/documents/AudioDocument_test.py

```
@@ -0,0 +1,45 @@
+"Are you threatening to quit?"
+"No, I'm threatening to be right. And when it breaks, I want it in writing that you chose this."
+"The feature ships, Sarah. That's final.\""""
+
+
+def test_AudioDocument():
+
+    document = AudioDocument(
+        id=uuid.uuid4(), name="audio-dummy-test", raw_data_location=""
+    )
+    with patch.object(AudioDocument, "create_transcript", return_value=TEST_TEXT):
+        l = list(document.read(chunk_size=64))
```

Ruff

Ambiguous variable name: `l`

(E741)